android/KMAPro/kMAPro/src/main/java/com/keyman/android/KmpInstallMode.java

```diff
@@ -0,0 +1,7 @@
+package com.keyman.android;
+
+public enum KmpInstallMode {
+  Silent,
```

Comment:
Can we have a description what the enums are for?

Reply:
I guess so, since you asked so nicely 🤣